index.html

```diff
@@ -329,7 +332,16 @@ <h1 class="about-section-h1">About Me</h1>
 
     </section>
   </main>
-
+  <div class="mobile-menu">
+    <svg class="iconx" width="24" height="24" viewBox="0 0 24 24" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+      <path fill-rule="evenodd" clip-rule="evenodd" d="M5.29289 5.29289C5.68342 4.90237 6.31658 4.90237 6.70711 5.29289L12 10.5858L17.2929 5.29289C17.6834 4.90237 18.3166 4.90237 18.7071 5.29289C19.0976 5.68342 19.0976 6.31658 18.7071 6.70711L13.4142 12L18.7071 17.2929C19.0976 17.6834 19.0976 18.3166 18.7071 18.7071C18.3166 19.0976 17.6834 19.0976 17.2929 18.7071L12 13.4142L6.70711 18.7071C6.31658 19.0976 5.68342 19.0976 5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929L10.5858 12L5.29289 6.70711C4.90237 6.31658 4.90237 5.68342 5.29289 5.29289Z"/>
+    </svg>
+    <ul>
+      <li><a href="#work-section">Portfolio</a></li>
+      <li><a href="#about-me">About</a></li>
+      <li><a href="#form-section">Contact</a></li>
+    </ul>
```

Comment:
- [ ] The dividers of your mobile menu elements are shorter than the provided [Figma template](![image](https://user-images.githubusercontent.com/12745474/131381056-6fb4dcdc-e2ea-4843-9172-bb09d73e962f.png)). We actually want the design to as much as possible look very close as the template design. Kindly ensure to make the dividers of the menu design match the provided template
- [ ]  Also, kindly ensure to adjust the `font size` and `spacing` between the menu items accordingly to match the provided template.

**Your mobile menu design:**
![image](https://user-images.githubusercontent.com/12745474/131381883-0c95b72c-116a-4e8e-8a79-769f6d16d618.png)


**The original menu design:**
![image](https://user-images.githubusercontent.com/12745474/131381213-94c8a99f-d2ca-4e97-98fc-763be4918b0d.png)